test/DynamoCoreWpf3Tests/NodeViewCustomizationTests.cs

```diff
@@ -304,16 +304,11 @@ public void WatchImageCoreContainsImage()
             var nodeView = NodeViewWithGuid("cf3ed4fb-f0a2-4dfe-89c1-11e8bbcfe80d");
                 // NodeViewOf<Dynamo.Nodes.WatchImageCore>();
 
-            var imgs = nodeView.ChildrenOfType<Image>();
+            var imgs = nodeView.PresentationGrid.ChildrenOfType<Image>();
 
-            // Starting from Dynamo 2.13, node view now comes with 
-            // images like node icon, lacing image etc
-            // As of Nov 2024, we have 8 images per NodeView
-            // Images are named for ease of use
-            // As of Dynamo 3.5, the number of images in a node view is 9 after the addition of the TransientImage
-            Assert.AreEqual(9, imgs.Count());
+            Assert.AreEqual(1, imgs.Count());
```

Comment:
so did we remove 8 images?

Reply:
Yes and no.  We removed the Extra Grid wrapper around the glyphs and collapse the images directly instead.  In this case however the test was always flawed.  It never tested if the actual image control for this node existed. The change also addresses that issue